### ISSUE TITLE
AsSSortedList returns something unsorted

### ISSUE THREAD

**Issue by mtorpey** [gap-packages/Semigroups#269]
`AsSSortedList` returns something that isn't an SSortedList, for a transformation semigroup created from a smallsemi semigroup via `AsSemigroup`:

```gap
gap> LoadPackage("smallsemi", false);;
gap> S := AsSemigroup(IsTransformationSemigroup, SmallSemigroup(5, 11));;
gap> IsSSortedList(AsSSortedList(S));
false
```

If you construct the same semigroup "normally" (using generators) it doesn't suffer from this problem.  So I expect this is something to do with `AsSemigroup`.

**Comment by james-d-mitchell** [comment]
This seems to be a bug in Smallsemi, in the `IsomorphismTransformationSemigroup` method in `small.gi` of Smallsemi are the lines:

    elts:=List(S, right);
    SetAsSSortedList(T, elts);

So, the elements of the range of `IsomorphismTransformationSemigroup` are set to be the images of the set of elements of the small semigroup, which is clearly not valid. I will resolve this in Smallsemi, and then close this issue.

**Comment by james-d-mitchell** [comment]
I have resolved this by releasing [Smallsemi 0.6.11](http://www-groups.mcs.st-andrews.ac.uk/~jamesm/smallsemi.php). 

There were some more issue with the tests for Smallsemi, which I have also resolved in Smallsemi 0.6.11 and commits 09380b1 and 376c210 in Semigroups. I think this is now resolved.

**Comment by wilfwilson** [comment]
Seems fine to me now.
